src/entities/trades/uniswap-v2/constants.ts

```diff
@@ -1,11 +1,21 @@
 import { ChainId } from '../../../constants'
-import { DXD, Token, WETH, WMATIC, WXDAI } from '../../token'
+import { DXD, Token, WBNB, WETH, WMATIC, WXDAI } from '../../token'
 
 // a list of tokens by chain
 type ChainTokenList = {
   readonly [chainId in ChainId]: Token[]
 }
 
+export const ETH: Record<number, Token> = {
+  [ChainId.BSC_MAINNET]: new Token(
+    ChainId.BSC_MAINNET,
+    '0x2170Ed0880ac9A755fd29B2688956BD959F933F8',
+    18,
+    'ETH',
+    'Binance-Peg Ethereum'
+  ),
+}
+
```

Comment:
What's the difference between this ETH and WETH? My understanding is Ether is on Ether on Mainnet, and true L2's like Arbitrum and Optimism. However, on sidechains such as Gnosis, Polygon PoS, and BNB, there's an IOU claim represented in WETH.

Reply:
ye... that token is Binance-Peg. My bad. I'll remove it.